image/image.go

```diff
@@ -52,7 +53,7 @@ func (i *genericImage) IntendedDockerReference() string {
 // NOTE: It is essential for signature verification that Manifest returns the manifest from which BlobDigests is computed.
 func (i *genericImage) Manifest() ([]byte, string, error) {
 	if i.cachedManifest == nil {
-		m, mt, err := i.src.GetManifest([]string{manifest.DockerV2Schema1SignedMIMEType, manifest.DockerV2Schema1MIMEType})
+		m, mt, err := i.src.GetManifest(i.supportedDestMIMEType)
```

Comment:
Like this, wouldn‘t a `copy docker:… dir:…` break by passing a nil slice here?

Considering the `requestedManifestMIMETypes` approach above, it would make sense for `image.FromSource` to replace a `nil` parameter (=“I don’t know or care”) by this hard-coded list of MIME types that `getParsedManifest` supports).